src/electron/electron/handler.cljs

```diff
@@ -244,7 +244,7 @@
                                   (.toString (.readFileSync fs txid-path)))]
           (reader/read-string sync-meta))))
     (catch :default e
-      (js/console.debug "[read txid meta] #" root (.-message e)))))
+      (logger/error "[read txid meta] #" root (.-message e)))))
```

Comment:
Non-blocking: debug?

Reply:
This should be an error. As it fails to read a file in the current graph folder even if it exists.